charts/generic-service/templates/ingress.yaml

```diff
@@ -24,17 +25,25 @@ metadata:
 spec:
   tls:
   - hosts:
-    - {{ .Values.ingress.host }}
+      - https-example.foo.com
+    secretName: testsecret-tls
```

Comment:
this doesnt look right?

Reply:
yep good spot! left the example in that i used to get the spacing right